Phases/Assessments/README.md

```diff
@@ -0,0 +1,131 @@
+
+Learners may choose to complete an assessment at any time during the phase, but, there are two rules:
+
+1. An assessment may only be attempted *twice*.
+2. A learner must *complete and pass* the assessment no later than the 8th week after they begin the phase.
+
+## The Process
+
+**Note:** This process is identical *regardless* of which phase assessment is being attempted.
+
+When a learner feels ready to attempt an assessment, a Moderator will manually assign them to a phase assessment. The lead coach for the project will be an SEP, and learners should have few or no meetings to attend during that week.
+
+The Moderator will send Learners the link to their assessment challenge on Monday morning. Learners will then have **2.5 days** to work on their solution and complete as much as they can. Learners _must submit_ their assessment solution/artifact by **1pm on Wednesday**.
+
+Evaluations and interviews start after lunch on Wednesday and continue until they are done, ideally before 6pm on Friday.
+
+After the evaluation and interviews have been completed, the results will be sent to Membership to make a decision and inform the learner whether they passed.
+
+### Requesting an Early Assessment
+
+If you would like to take the assessment _before_ the 8th week of your phase, you can request an early assessment from the Moderator. The Moderator will attempt to respond to these requests as best they can, but there is no guarantee that every request will be immediately met.
+
+## Preparing for the Assessment
+
+Make sure to read the appropriate assessment overview page for the assessment you'll be taking ([1](./Phase_1.md), [2](./Phase_2.md), or [3](./Phase_3.md)). This is where you'll find the most relevant information about the assessment to guide your preparation.
+
+In addition, we recommended that you take following steps to lessen the effects of test anxiety.
+
+- Before the assessment...
+  - Study the relevant skills and take notes for yourself.
+  - Organize the code you've written for easy reference (the assessment is open-book).
+  - Get plenty of sleep. Exercise. Eat well. Take care of yourself. :)
+- During the assessment...
+  - Make sure to read the instructions very closely.
+  - Don't expect to complete everything, just do as much as you can.
+  - Focus on the areas that play to your strengths first, then do the hard stuff. Feel free to jump around.
+  - Take frequent, short breaks to rest your brain.
+
+## Getting Support
+
+As they are working on their assessment code, learners can ask their lead coach (who is an SEP) for support. For example, learners may have to ask clarifying questions about the goal or its specs, or may need help debugging strange environment or git issues.
```

Comment:
`learners can ask their lead coach (who is an SEP) for support` how does this change for an interview? You cannot get support from an interviewer on a take home exercise. I think we need to redefine support for these interviews.